README.md

```diff
@@ -1,8 +1,24 @@
-Best viewed on the website: https://docs.ankiweb.net
+# Anki Manual
 
-This manual is licensed under the CC BY-SA 4 license:
+This repository contains the source of [Anki's user manual](https://docs.ankiweb.net).
 
-https://creativecommons.org/licenses/by-sa/4.0/
+## License 
 
-Pull requests are welcome. Unless you state otherwise, any pull requests you
-submit will be licensed under the above license. Please see [CONTRIBUTING](CONTRIBUTING.md) for more details.
+This manual is licensed under the [CC BY-SA 4 license](https://creativecommons.org/licenses/by-sa/4.0/).
+
+## Contributing
+
+Contributions are welcome! Please see the [issue tracker](https://github.com/ankitects/anki-manual/issues) for things you can start working on. For details on local development, see [CONTRIBUTING](CONTRIBUTING.md).
+
+Unless stated otherwise, any pull requests you
+submit will be licensed under the [CC BY-SA 4 license](https://creativecommons.org/licenses/by-sa/).
+
+## Style Guidelines
+
+The following conventions are not consistently applied in the manual. If you find such text, feel free to change it.
+
+- Use double quotes, `"`, instead of single quotes, `'`.
+- For emphasis, when defining anything or using words as words, use italics, `_`.
```

Comment:
Using words as words?

Reply:
If you're talking about a word rather than what the word refers to, it's called "using words as words".

Reply:
It's my first time hearing that! Can you provide an example other than defining a word where it would make sense? Would you be comfortable with altered wording like the following instead?

"For emphasis, such as when defining words, use italics ..."

Reply:
> Can you provide an example other than defining a word where it would make sense?

"To search for either _dog_ and _cat_, use `dog or cat`"

> Would you be comfortable with altered wording like the following instead?
> 
> "For emphasis, such as when defining words, use italics ..."

Sure. Do we keep the other suggestion too?

Reply:
Sorry, I'm not following. Do you mean "words on words"? I was suggesting we drop that, as I at least found it confusing, so I'm assuming some other people may too. Or do you mean something else?

Reply:
See [this](https://www.chicagomanualofstyle.org/qanda/data/faq/topics/NoneoftheAbove/faq0012.html) if you're confused what "words as words" mean.

As for where this guideline will apply for example, if you're talking about searching for the word "dog" in Anki, instead of putting double quotes around it, you'd use _italics_. That's the example I gave earlier.

I can write this as a seperate point with an example if this was confusing.

Reply:
I understand what you're saying, I just think that "words as words" is not particularly clear. I'd prefer you use another example instead of that particular phrase, or wait for someone else to chime in to see if they disagree with me.

Reply:
Check the latest commit.

(I couldn't find anything better at that time so decided to go with that.)

Reply:
Thank you!